### ISSUE TITLE
Wget already installed error

### ISSUE THREAD

**Issue by grant-h** [cosmo0920/homebrew-mingw_w64#10]
When installing after tapping I get this error:

```
=> Pouring gcc49-4.9.3.el_capitan.bottle.2.tar.gz
🍺  /usr/local/Cellar/gcc49/4.9.3: 1,160 files, 162.3M
Warning: homebrew/versions/gmp4-4.3.2 already installed
Warning: homebrew/versions/libmpc08-0.8.1 already installed
Warning: homebrew/versions/mpfr2-2.4.2 already installed
Warning: homebrew/versions/cloog018-0.18.0 already installed
Warning: homebrew/versions/isl011-0.11.1 already installed
Error: wget-1.17 already installed
To install this version, first `brew unlink wget`
```

I can reproduce this error locally by manually running `brew install wget`. Not sure why this is happening.

To fix, I ran `brew unlink wget && brew install wget`.

Is this an issue with this package or upstream? I don't know why it presented me with an error instead of just updating.

**Comment by cosmo0920** [comment]
> When installing after tapping

Unfortunately, I have no idea why this error happened.

Could you show complete reproduce steps?

Just tapping this repository? Or executing `brew install install-mingw-w64`?

**Comment by cosmo0920** [comment]
> I can reproduce this error locally by manually running brew install wget.

Really? I got the following result:

``` log
% brew install wget
Warning: wget-1.17.1 already installed
```

It shows not error but warning.
Umm..., it's really curious.... :confounded:

**Comment by grant-h** [comment]
Very strange! I had `wget-1.17` and it wanted to upgrade to `wget-1.17.1`. For some reason this Error'd out within `brew install-mingw-w64`. I manually ran `brew unlink wget && brew install wget` to resolve this issue. This doesn't seem like an issue with your package. You didn't do anything wrong, but I figured that brew wouldn't fail to install a dependency like that.

We could probably close this issue.
